cmd/kubectl-gadget/biotop.go

```diff
@@ -0,0 +1,274 @@
+// Copyright 2019-2021 The Inspektor Gadget authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package main
+
+import (
+	"encoding/json"
+	"fmt"
+	"os"
+	"strconv"
+	"strings"
+	"sync"
+	"time"
+
+	"github.com/spf13/cobra"
+	"golang.org/x/term"
+
+	"github.com/kinvolk/inspektor-gadget/cmd/kubectl-gadget/utils"
+	"github.com/kinvolk/inspektor-gadget/pkg/gadgets/biotop/types"
+)
+
+var (
+	nodeBioStats map[string][]types.Stats
+	mutex        sync.Mutex
+	finished     bool = false
+)
+
+var (
+	// arguments
+	printInterval int = 1
+
+	// flags
+	maximumRows int
+	sortTypeStr string
+	sortType    types.SortBy
+)
+
+var biotopCmd = &cobra.Command{
+	Use:   "biotop [interval]",
+	Short: "Trace reads and writes by file, with container details",
+	RunE: func(cmd *cobra.Command, args []string) error {
+		var err error
+
+		nodeBioStats = make(map[string][]types.Stats)
+
+		if len(args) == 1 {
+			printInterval, err = strconv.Atoi(args[0])
+			if err != nil {
+				return fmt.Errorf("%q is not a valid value for printInterval", args[0])
+			}
+		}
+
+		config := &utils.TraceConfig{
+			GadgetName:       "biotop",
+			Operation:        "start",
+			TraceOutputMode:  "Stream",
+			TraceOutputState: "Started",
+			CommonFlags:      &params,
+			Parameters: map[string]string{
+				"max_rows":      strconv.Itoa(maximumRows),
+				"printInterval": strconv.Itoa(printInterval),
+				"sort_by":       sortTypeStr,
+			},
+		}
+
+		startoutput()
+
+		if err := utils.RunTraceStreamCallback(config, hook); err != nil {
+			return fmt.Errorf("error running trace: %w", err)
+		}
+
+		return nil
+	},
+	SilenceUsage: true,
+	PreRunE: func(cmd *cobra.Command, args []string) error {
+		var err error
+		sortType, err = types.ParseSortBy(sortTypeStr)
+		if err != nil {
+			return err
+		}
+
+		return nil
+	},
+	Args: cobra.MaximumNArgs(1),
+}
+
+func init() {
+	biotopCmd.Flags().IntVarP(&maximumRows, "maximumRows", "r", 20, "Maximum rows to print")
+	biotopCmd.Flags().StringVarP(&sortTypeStr, "sort", "", "all", "Sort column")
```

Comment:
What about showing the supported values with `strings.Join(types.SortBySlice, ",")`?